runtime/jcl/j9jcl.tdf

```diff
@@ -700,7 +700,7 @@ TraceExit=Trc_JCL_J9SigUsr2Startup_Unavailable noEnv Overhead=1 Level=1 Template
 TraceExit=Trc_JCL_J9SigUsr2Startup_Failure noEnv Overhead=1 Level=1 Template="J9SigUsr2Startup (failed to start)"
 TraceExit=Trc_JCL_J9SigUsr2Startup_Exit noEnv Overhead=1 Level=1 Template="J9SigUsr2Startup"
 
-TraceEntry=Trc_JCL_J9SigUsr2Shutdown_Entry noEnv Overhead=1 Level=1 Template="J9SigUsr2Shutdown"
-TraceExit=Trc_JCL_J9SigUsr2Shutdown_Exit noEnv Overhead=1 Level=1 Template="J9SigUsr2Shutdown"
```

Comment:
Existing tracepoints cannot be deleted (if no longer needed they can be marked "Obsolete", but I don't think you meant that).